Competition2017/tests/competition/subsystems/drive/DriveTestBase.java

```diff
@@ -53,10 +53,60 @@ public void verifyDriveNegative() {
     public void setDriveEncoderDistances(double left, double right) {
         ((MockCANTalon)drive.leftDrive).setPosition(left);
         ((MockCANTalon)drive.rightDrive).setPosition(right);
+        
+        drive.updatePeriodicData();
+        pose.updatePeriodicData();
     }
     
     public void driveDeltaEncoderDistances(double left, double right){
```

Comment:
Is this in ticks? Now that there's an `inches` version, I think this one should have the units in the name too.